src/View/UIViewTrait.php

```diff
@@ -13,11 +13,16 @@
     /**
      * Initialization hook method.
      *
+     * ### Options:
+     *
+     * - `layout` to pass your layout, or it will use the plugin one
+     *
+     * @param array $options
      * @return void
      */
-    public function initializeUI()
+    public function initializeUI(array $options = [])
     {
-        if (empty($this->layout) || $this->layout === 'default') {
+        if (empty($options['layout'])) {
```

Comment:
yes @jadb I thought like you, but I didn't found it meaningful. take another look, I may not right